tensorflow/python/kernel_tests/random/random_binomial_test.py

```diff
@@ -207,6 +208,24 @@ def moment(self, i):
         )
         self.assertAllLess(z_scores, z_limit)
 
+  def testStatelessDtypes(self):
+    # Test case for GitHub issue 59160
+    shape_0 = constant_op.constant(1, dtype=dtypes.int64) #125091515651
```

Comment:
what does "#125091515651" mean here?

Reply:
do we need to test other types here? @cantonios

Reply:
@vufg The `125091515651` was the initial "large" number I tried to test. Then I noticed that any number with int64 type will trigger the crash (before the fix). Therefore I changed the value from 125091515651 to 1 to make test simple.